src/frontend-helper.ts

```diff
@@ -112,3 +129,51 @@ const setImageStatus = (images: Image[], status: IMAGE_STATUS) => {
     );
   });
 };
+
+class policyStatusByAdGroup {
+  // adGroupIssues[adGroupId][filename]
+  static adGroupIssues: {
```

Comment:
Remove commented-out code?

If it's for documentation purposes, we could describe the keys in the type below